tools/debian/control

```diff
@@ -77,7 +77,6 @@ Package: cockpit-docker
 Architecture: all
 Depends: ${misc:Depends},
          docker.io (>= 1.3.0) | docker-engine (>= 1.3.0),
-         python,
```

Comment:
> However, it is true, there is no such thing as debian atomic and python is only used for dealing with storage on atomic.

The cockpit-atomic-storage script is used everywhere.  Depending on what is installed, it will let you manage the storage or not.  In all cases it is used to monitor the storage usage of images.

Thus, it always needs to run, even on Debian and Ubuntu.

Let's make sure it works with Python 3.

Does it need to work with both Python 2 and 3?  I am not sure.  The "atomic" command itself supports that for some reason.